mvp.feature

```diff
@@ -191,295 +290,453 @@ Feature: User with no Author identity in the Authority Register for Personas see
     And they see that the correct authors have the expected value for Corresponding Author
     And they see that the Other Contributors are listed alphabetically by Surname
     And they see that the Other Contributors have the correct Role, Name and Institution
-    And the Next button is enabled
 
-    # Corresponding Author is not in fact present in the Datacite data, but probably will be soon
+  @417
+  Scenario: User navigates to <Contributor> tab
+    Given that the user verifies initial metadata for a Publication based on a Link
+    When they select the <Contributor> tab
+    Then they see that there are no Contributors registered
+    And a grid is shown with the fields
+      | Name           |
+      | Institution    |
+      | Corresponding  |
+      | Move up / down |
+      | Edit           |
+      | Delete         |
+    And a button to add an Author is enabled
+
+  @418
+  Scenario: User opens the add Author dialog
+    Given that the user navigates to <Contributor> tab
+    When they click the to add an Author
```

Comment:
`When they click the Button to add an Author` eller `When they click the Add Author Button` ?